.github/workflows/ci-cd-am-tele.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: AlertManager Telegram Integration CI
+  
+on:
+  push:
+    branches: [ master ]
+    paths:
+    - alertmanager-notifier-gateway/alertmanager-telegram-gateway/*
+  pull_request:
+    branches: [ master ]
+    paths:
+    - alertmanager-notifier-gateway/alertmanager-telegram-gateway/*
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    - uses: docker/build-push-action@v1
+      with:
+        path: alertmanager-notifier-gateway/alertmanager-telegram-gateway/
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+        repository: redhat-cop/alertmanager-telegram-gateway
```

Comment:
This doesn't seem to be an existing quay repo. Please submit issue to request one before we merge this PR:
https://github.com/redhat-cop/org/issues/new/choose

Reply:
https://github.com/redhat-cop/org/issues/199 submitted.